plugins/eslint/validateImports.js

```diff
@@ -26,7 +26,8 @@ function flagErrors(context, node, importPath) {
     if (
       path.dirname(absImportPath) === absModulePath || (
         absImportPath.startsWith(absModulePath) &&
-        path.basename(absImportPath) === 'index.js'
+        path.basename(absImportPath) === 'index.js' &&
+        path.basename(absFileDir) !== 'prebidServerBidAdapter'
```

Comment:
Is this still needed anymore, since we changed how the imports work for the PBS adapter?

Reply:
No need for this anymore
Removed changes